### PR TITLE
Update to angular 10

### LINKED ISSUE #341: please update to @swimlane/ngx-charts": "^14.0.0
This gives me compilation issues as there are now 2 major versions installed 

thanks

### PR DESCRIPTION
**Pull request by marjan-georgiev**
**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

Closes #341

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [ ] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by haeminn** [comment]
@marjan-georgiev 
According to the upgrade checklist, should we do this? `Angular now recommends the use of a tsconfig.base.json to help organize the various typings contexts (shared, unit tests, end to end tests, application, etc). ng update will migrate you automatically.`

File: projects/swimlane/ngx-graph/package.json [hunk 1]
```diff
@@ -37,7 +37,7 @@
     "rxjs": "6.x"
   },
   "dependencies": {
-    "@swimlane/ngx-charts": "^13.0.1",
+    "@swimlane/ngx-charts": "^16.0.1",
```
Comment (haeminn) [inline projects/swimlane/ngx-graph/package.json:40]: ngx-charts latest version is `16.0.0`

Comment (marjan-georgiev) [reply projects/swimlane/ngx-graph/package.json]: pushed

**Review by haeminn — APPROVED** [review]
lgtm

**Pull request merged**